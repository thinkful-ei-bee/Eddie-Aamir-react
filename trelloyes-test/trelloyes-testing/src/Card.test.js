import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';

describe('<Card />',()=>{
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title={'Card Title'} content={'Content...'}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
