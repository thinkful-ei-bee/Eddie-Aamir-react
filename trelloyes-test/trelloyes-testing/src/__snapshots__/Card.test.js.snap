// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> render the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    Card Title
  </h3>
  <p>
    Content...
  </p>
</div>
`;
